flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  description = "Tabry in Rust";
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  };
+  outputs = { nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+        let
+          pkgs = nixpkgs.legacyPackages."${system}";
+          tabry = pkgs.callPackage ./default.nix {};
+        in {
+          packages = {
+            default = tabry;
+          };
+          devShell = pkgs.mkShell {
+            packages = with pkgs; [
+              rustfmt
+              cargo
+              rustc
```

Comment:
I also use clippy, cargo-llvm-cov, and bacon

Reply:
Ah okay, are those baked into cargo commands? Does cargo handle pulling those, or do they need to be installed/maintained separately (for which they could be added to this list)? I'm not too familiar with the ecosystem yet

Reply:
They are installed separately, although clippy and llvm-cov are run through cargo. When not using nix, clippy and cargo-llvm-cov are usually installed using rustup I believe, and bacon is installd using 'cargo install bacon', but there are probably nix derivations for all of them. Bacon is really nice, it watches your files and runs tests, builds, or runs clippy (linter)... you don't have to tel it which files or remember any command line arguments, you just type 'bacon'.